### PR TITLE
Mu 17

### PR DESCRIPTION
**Pull request by Staroz**
Please, review this PR, I know that one YML file is missing, which is a problem whit the Playwright version, and I am trying fix to it.

### CODE REVIEW

File: cypress/fixtures/credentials.json
Comment (sphinix27) [inline cypress/fixtures/credentials.json:12]: Some updates are missing from this example file

**Review by sphinix27 — COMMENTED** [review]
LGTM, I'll wait until the other PR is merged before approving this one

**Comment by Staroz** [comment]
Hi, I've merged PR mu_16, please, review this PR to pass the next level thanks.

**Review by sphinix27 — APPROVED** [review]
LGTM

**Pull request merged**
